packages/account/src/Sections/Profile/FinancialAssessment/financial-assessment.jsx

```diff
@@ -304,6 +342,7 @@ class FinancialAssessment extends React.Component {
                             other_instruments_trading_frequency,
                         }),
                     }}
+                    enableReinitialize={true}
```

Comment:
would resetForm work instead? https://github.com/formium/formik/issues/446

Reply:
Are there downsides to enableReinitialize? The personal details section at the moment utilises this for the form as well.